src/PlaywrightEnvironment.ts

```diff
@@ -133,7 +133,9 @@ class PlaywrightEnvironment extends NodeEnvironment {
     this.global.browser = await getBrowserPerProcess(playwrightInstance, config)
     this.global.context = await this.global.browser.newContext(contextOptions)
     this.global.page = await this.global.context.newPage()
-    this.global.page.on('pageerror', handleError)
+    if (config.exitOnPageError) {
```

Comment:
It should be distructed from here:
https://github.com/playwright-community/jest-playwright/blob/8ce19bc9adf87cb5380c80ebd8f2e5cda51a8c5c/src/PlaywrightEnvironment.ts#L95

Reply:
Thanks @mmarkelov. Fixed.